relayer/utils/__init__.py

```diff
@@ -0,0 +1,3 @@
+def get_elapsed_time_in_milliseconds(start_time, end_time):
+    elapsed_time = end_time - start_time
+    return elapsed_time.microseconds / 1000.0 + elapsed_time.seconds * 1000
```

Comment:
So te code divides microseconds by a thousand to convert them to milliseconds, and multiplies the seconds by 1000 to get milliseconds again, those are added together to get the total number of milliseconds.